python/cudf/cudf/core/column/string.py

```diff
@@ -782,6 +782,70 @@ def contains(
             )
         return self._return_or_inplace(result_col)
 
+    def like(self, pat: str, esc: str = None) -> SeriesOrIndex:
+        """
+        Test if a like pattern matches a string of a Series or Index.
+
+        Return boolean Series or Index based on whether a given pattern
```

Comment:
We should indicate somewhere that this behavior is meant to match the SQL `LIKE` operator (and possibly specify the dialect that it matches) -- it's not necessarily obvious that this is the same, and users will want to know that it acts the same way.

Reply:
Unfortunately, there are several implementations of SQL LIKE with different features and behaviors.
I would rather document the behavior as it works (and will be supported) rather then to promise it works like any specific implementation of SQL.

Reply:
Here is one summary from the referenced issue
https://stackoverflow.com/questions/712580/list-of-special-characters-for-sql-like-clause